bibliopixel/animation/runner.py

```diff
@@ -62,6 +66,10 @@ def __init__(self, *, amt=1, fps=0, sleep_time=0, max_steps=0,
         self.threaded = threaded
         self.flat_out = flat_out
         self.main = load.code(main)
+        if repeats is not None:
```

Comment:
This will need to be made clear that this ONLY works if the animation supports it. Otherwise this does nothing and the animation runs forever.

Reply:
I'm not sure what you mean?  This translates exactly into the old values.  Nothing changes...